astropy/io/votable/volint.py

```diff
@@ -3,14 +3,18 @@
 Script support for validating a VO file.
 """
 
+from ... import __version__
```

Comment:
Version is *generated* during build. I don't think this is the proper way to access it?